MeshKernelNET/test/MeshKernelNETCoreTest/Api/MeshKernelTest.cs

```diff
@@ -1519,5 +1519,38 @@ public void ContactsComputeWithPointsThroughAPI()
             }
         }
 
+        [Test]
+        public void GetSelectedVerticesInPolygonThroughAPI()
+        {
+            using (var mesh = GenerateRegularGrid(4, 4, 1, 1))
+            using (var api = new MeshKernelApi())
+            {
+                var id = 0;
```

Comment:
not possible to declare as const int in the try block?

Reply:
we need the id for later deallocation in the finally block